.github/workflows/integration.yml

```diff
@@ -61,7 +61,7 @@ jobs:
       # Prerequisites
       - uses: ./.github/actions/restore-node
         with:
-          node-version: 16.x
+          node-version: 18.x
```

Comment:
Didn't realize we were still on Node 16 here!

Reply:
This may be breaking the "getting-started" integration test.

cc @LuqiPan who is currently working on updating this test.

Reply:
This is actually what we need to do: https://github.com/Agoric/agoric-sdk/pull/8829/files#diff-82454b2fc887e9985b9348b8f3bca03d2f839645a4c3ba1f3f850014d7da5c8bL60

Reply:
Ah yeah, pinning this at `18.18` might be the way to unblock for now. I'd like to get our dapps to handle node 18.19, but that's another story.